lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -0,0 +1,66 @@
+module Runners
+  class Processor::MetricsFileInfo < Processor
+    Schema = _ = StrongJSON.new do
+      # @type self: SchemaClass
+
+      let :runner_config, Schema::BaseConfig.base
+      let :issue, object(
+        line_of_code: integer?,
+        last_commit_datetime: integer
+      )
+    end
+
+    def analyzer_version
+      @analyzer_version ||= extract_version!("wc")
+    end
+
+    # This analyser use git metadata (.git/).
+    def use_git_metadata_dir?
+      true
+    end
+
+    def analyze(changes)
+      Results::Success.new(guid: guid, analyzer: analyzer).tap do |result|
+        changes.changed_paths.each do |path|
+          result.add_issue(generate_issue(path))
+        end
+      end
```

Comment:
[suggestion] How about this refactoring?

```suggestion
      Results::Success.new(
        guid: guid,
        analyzer: analyzer,
        issues: changes.changed_paths.map { |path| generate_issue(path) },
      )
```

Reply:
fix: 16f13165c9ec32eec54ed081ba7251db54d7e759